org.civicrm.reqangular/test/services/api.spec.js

```diff
@@ -37,13 +32,15 @@ define([
         };
 
         beforeEach(function () {
-          promise = expectAndSendGET(returnValue);
+          expectAndSendGET(returnValue)
+            .then(function (response) {
+              expectedResponse = response;
+            });
```

Comment:
According to the styleguide we are missing the "done" execution:

```js
beforeEach(function (done) {
  expectAndSendGET(returnValue)
    .then(function (response) {
      expectedResponse = response;
    })
    .finally(done);
  // a space here wouldn't be bad either
  $httpBackend.flush();
});
```

Reference: https://compucorp.atlassian.net/wiki/spaces/PCHR/pages/121371880/AngularJS#AngularJS-Testingasynchronousmethods

Reply:
Can we do this for all the other async tests?

Reply:
This test is not really "async" since it utilises Angular methods. It would have been if it utilised "setTimeout" inside for example, something that is outside Angular's control.

Reply:
As per the Styleguide we don't add spaces between methods. The only exception could be the `resolve()` method call which kinda behaves like `return`.